src/js/websocket.js

```diff
@@ -16,7 +16,12 @@
  var net = require('net');
  var util = require('util');
  var EventEmitter = require('events').EventEmitter;
- var tls = require('tls');
+ var tls;
+ try {
+   tls = require(...);
```

Comment:
In my previous comment the `...` was meant to represent that you need to replace it with the `'tls'` string. This is a syntax error currently.  It should be: `require('tls')`